src/ocean/net/device/Berkeley.d

```diff
@@ -1353,7 +1355,14 @@ deprecated public class IPv4Address : Address
         static uint parse(istring addr)
         {
                 char[64] tmp;
-                return ntohl(inet_addr(toStringz(addr, tmp)));
+                if ((addr.length + 1) > tmp.length)
```

Comment:
I am a bit dubious myself TBH. But 90% of the affected code is old (everything but `DropPriviledgesExt`, which we could look into deprecating, is Tango AFAICS).